src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_31">
+      <div class="calendar__block  calendar__block--start_sun"></div>
+      <div class="calendar__block"></div>
```

Comment:
Better rename all, this way it would be more obvious.
```suggestion
      <div class="calendar__day"></div>
```